### PR TITLE
Fix local & CI builds

### PR DESCRIPTION
**Pull request by allan-on**
- Updated `kujaku` dependency to resolve from GH packages
- Added GH Properties user and key env values to the CI config
- Removed Bintray repositories
- Updated `id.zelory:compressor` dependency
- Updated FHIR dependency

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -68,10 +64,10 @@ configure(allprojects) { project ->
     buildscript {
         repositories {
             google()
-            jcenter()
             mavenCentral()
             mavenLocal()
             maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
+            jcenter()
```
Comment (ndegwamartin) [inline build.gradle:70]: Do we still need JCenter here?

File: build.gradle [hunk 2]
```diff
@@ -54,10 +52,8 @@ allprojects {
         maven { url 'https://maven.fabric.io/public' }
         maven { url "https://s3.amazonaws.com/repo.commonsware.com" }
         maven { url "https://s3.amazonaws.com/repo.commonsware.com" }
-        maven { url "https://dl.bintray.com/ona/rdt-capture" }
-        maven { url "http://dl.bintray.com/ona/kujaku" }
-        maven { url 'https://dl.bintray.com/ibm-watson-health/ibm-fhir-server-releases'}
         maven{  url "https://plugins.gradle.org/m2/" }
+        jcenter()
```
Comment (ndegwamartin) [inline build.gradle:56]: Do we still need JCenter here?

Comment (allan-on) [reply build.gradle]: No, we don't. I've removed the entry

File: build.gradle (hunk 1 above)
Comment (allan-on) [reply build.gradle]: No, we don't. I've removed the entry

**Review by ndegwamartin — DISMISSED** [review]

**Comment by allan-on** [comment]
The FormsUtils test ignored in this PR will be fixed in this [issue](https://github.com/opensrp/opensrp-client-chw-core/issues/433)

**Review by ndegwamartin — APPROVED** [review]

**Pull request merged**